docs/reference/errors-and-warnings/NU5046.md

```diff
@@ -0,0 +1,58 @@
+---
+title: NuGet Error NU5046
+description: NU5046 Error code
+author: dominoFire
+ms.author: feaguila
+ms.date: 9/4/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5046
+---
+
+# NuGet Error NU5046
+
+<pre>The icon file 'Icon.png' does not exist in the package.</pre>
+
+
+# Issue
+
+NuGet is unable find the local file that is marked as the package icon.
+
+
+# Solution
+
+- Make sure that the file that is marked as the package icon exists and it is readable.
+- Ensure that the file is referenced in the nuspec or in the project file.
+  * When you are creating a package from a nuspec file, make sure to include the icon file in the `<files/>` section:
+
+    ```xml
+    <package>
```

Comment:
We want to encourange customers to use our pack targets, rather than using a nuspec file. So, the MSBuild example should be first. Same with the other file.